src/ostorlab/agent/mixins/agent_mq_mixin.py

```diff
@@ -50,15 +51,18 @@ def __init__(
             self._get_channel, max_size=64, loop=self._loop
         )
 
-    async def _get_connection(self) -> aio_pika.Connection:
-        return await aio_pika.connect_robust(url=self._url, loop=self._loop)
+    async def _get_connection(self) -> aio_pika.abc.AbstractRobustConnection:
+        kwargs = {"fail_fast": "false"}
+        return await aio_pika.connect_robust(url=self._url, loop=self._loop, **kwargs)
```

Comment:
```suggestion
        return await aio_pika.connect_robust(url=self._url, loop=self._loop, fail_fast=False)
```

Why is false a string and not a bool?

Reply:
They didn't support flexible parsers before, but this has been fixed in the current version.
https://github.com/mosquito/aiormq/commit/1cdfb06d58d6be91028207b62d9a47653da4d199

Good catch. I will update it to false instead.